www/assets/css/theme.css

```diff
@@ -1,28 +1,59 @@
+/* ╭─────────────────────────────────────────────────────────╮
+ * │                         Palette                         │
+ * ╰─────────────────────────────────────────────────────────╯
+ */
+:root {
+  --bg-main:   #fff6f6;  /* <body> background color */
+  --fg-main:   #333;     /* main text color */
+  --fg-em:     #d63;     /* mostly used by titles and <a> */
+
+  --fg-banner: #544;     /* also by "ergol rocket" and <code> */
+  --bg-banner: #efefef;  /* also used by tables of content */
+
+  --bg-root:   #ccc;     /* <html> background color */
+  --bg-em:     #d634;    /* em color, but with less opacity */
+  --bg-key:    #dfdfdf;  /* mostly used by <kbd> */
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --bg-main:   #14141a;
+    --fg-main:   #ccc;
+    --fg-em:     #f96;
+
+    --fg-banner: #cbb;
+    --bg-root:   #08080d;
+    --bg-em:     #f964;
```

Comment:
Any reason for not using `--fg-link` instead of `--fg-em` ?

As previously asked, I would really prefer **not** to use `-em` for both `--fg` and `--bg` if they don’t refer to the same element.  About any other name would be fine by me at this point. :-)

Reply:
> As previously asked

That’s my bad, I didn’t see you added other comments after your original response.

At first I didn’t want to use `--fg-link` ’cause we also use it for headings, but we’re also using `--bg-banner` in tables of content, so I don’t think it’s that big a deal.